tests/services/test_criteria_matcher.py

```
@@ -16,7 +14,13 @@ def mock_stat():
     """Fixture to create a mock stat_result object."""
 
     def _mock_stat(
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_mock_stat`

(ANN202)